src/bigint.cpp

```diff
@@ -437,6 +435,21 @@ bool mul_u64_overflow(uint64_t op1, uint64_t op2, uint64_t *result) {
 }
 #endif
 
+static inline void _twos_complement_inplace(BigInt *bn) {
```

Comment:
No explicit `inline` unless:
 * you need it so that you can put the function in a header file
 * you have a benchmark to demonstrate that it's faster and not too much code size increase
 * the function semantics depends on always being inlined. in this case you would need a compiler-specific attribute anyway.

Also, no need for leading underscore on static functions.

Reply:
Okay, I will change the signature.